go/internal/ocxcli/doctor_probes4.go

```
@@ -0,0 +1,362 @@
+		nonceFn = doctorManagementNonce
+	}
+	nonce, err := nonceFn()
+	if err != nil {
+		return nil, err
+	}
+	now := time.Now()
+	if r.Now != nil {
+		now = r.Now()
+	}
+	expires := now.Add(10 * time.Second).UnixMilli()
+	cap := managementauth.CreateLocalManagementReadCapability(r.Runtime.AttestationSecret, nonce, http.MethodGet, path, r.Runtime.PID, r.Runtime.Port, expires)
```

golangci-lint (2.13.2)

[error] variable cap has same name as predeclared identifier

(predeclared)